compiler/src/vm/builtin_functions.rs

```diff
@@ -135,7 +135,7 @@ impl Vm {
 
     fn print(&mut self, args: Vec<Value>) -> Result<Value, String> {
         destructure!(args, [Value::Text(message)], {
-            println!("{message:?}");
+            log::info!("{message:?}");
             Ok(Value::nothing())
         })
```

Comment:
I think this broke, maybe during a merge?

Reply:
What's the problem here? This seems fine to me

Reply:
Interesting, this was just showing me a different diff… Then it LGTM